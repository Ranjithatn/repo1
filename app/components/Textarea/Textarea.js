import React from "react";

const Textarea = ({
  id,
  className,
  value,
  onChange,
  cols,
  rows,
  disabled = false
}) => {
  return (
    // <div className="field">
    //   <div className="control">
    <textarea
      className={"textarea" + (className ? " " + className : "")}
      id={id ? id : ""}
      cols={cols ? cols : 30}
      rows={rows ? rows : 5}
      value={value}
      onChange={onChange}
      disabled={disabled}
    />
    //   </div>
    // </div>
  );
};

export default Textarea;
