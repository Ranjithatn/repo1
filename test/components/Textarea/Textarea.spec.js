import React from "react";
import toJson from "enzyme-to-json";
import Textarea from "../../../app/components/Textarea/Textarea";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });

describe('<Textarea />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <Textarea  />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });

});
