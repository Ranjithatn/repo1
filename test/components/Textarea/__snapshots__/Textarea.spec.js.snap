// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Textarea /> should render without crashing 1`] = `
<Textarea>
  <textarea
    className="textarea"
    cols={30}
    disabled={false}
    id=""
    rows={5}
  />
</Textarea>
`;
